src/loaders/MaterialLoader.js

```diff
@@ -77,8 +77,13 @@ Object.assign( MaterialLoader.prototype, {
 		if ( json.colorWrite !== undefined ) material.colorWrite = json.colorWrite;
 		if ( json.wireframe !== undefined ) material.wireframe = json.wireframe;
 		if ( json.wireframeLinewidth !== undefined ) material.wireframeLinewidth = json.wireframeLinewidth;
+		if ( json.wireframeLinecap !== undefined ) material.wireframeLinecap = json.wireframeLinecap;
+		if ( json.wireframeLinejoin !== undefined ) material.wireframeLinejoin = json.wireframeLinejoin;
+		if ( json.skinning !== undefined ) material.skinning = json.skinning;
+		if ( json.morphTargets !== undefined ) material.wireframe = json.morphTargets;
```

Comment:
Not sure if I understand correctly but is this line right?
Why wireframe is set when morphTargets is not undefined?